README.markdown

```diff
@@ -99,6 +100,18 @@ STP relies on : boost, flex, bison and minisat. You can install these by:
 $ sudo apt-get install cmake bison flex libboost-all-dev python perl minisat
 ```
 
+If your distribution does not come with minisat, STP maintains an updated fork; it can be built as follows:
+
+```
+$ git clone https://github.com/stp/minisat
+$ cd minisat
+$ mkdir build && cd build
+$ cmake ..
+$ make
+$ sudo make install
+$ sudo ldconfig
```

Comment:
`ldconfig` only applies to Linux, right? maybe: `command -v ldconfig && sudo ldconfig`

Reply:
I don't normally install packages like this system wide (ever), so I'm not sure (I know what `ldconfig` does, but I am not sure how essential it is here).

I can add the test if we prefer (I only don't prefer this as it might not be clear to people what `command -v <thing>` does).

Reply:
How about a note after it that says, "The final command may not be necessary on some systems."

Reply:
Done!